src/components/SelectField/SelectField.module.css

```diff
@@ -72,7 +54,10 @@
 .select-field__icon {
   position: absolute;
   top: 50%;
+  height: var(--eds-size-3);
+  width: var(--eds-size-3);
   transform: translateY(-50%);
   right: var(--eds-size-1);
   pointer-events: none; /* 1 */
+  fill: var(--eds-theme-color-icon-brand-primary);
```

Comment:
`color` and `size` props should be updated in `SelectField.tsx` instead, when calling `<Icon />`